lect_07/dags/upload_sales_to_gcs.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+from general.custom_config import cache_folder
+from upload_to_gcs import upload_folder
+
+DEFAULT_ARGS = {
+    'depends_on_past': False,
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 0,
+    'retry_delay': 60,
+}
+
+sales_dir = os.path.join(cache_folder, 'sales', '{{ ds }}')
+year = "{{ execution_date.strftime('%Y') }}"
+month = "{{ execution_date.strftime('%m') }}"
+day = "{{ execution_date.strftime('%d') }}"
+
+
+dag = DAG(
+    dag_id='upload_sales_to_gcs',
+    start_date=datetime(2022, 8, 1),
+    end_date=datetime(2022, 8, 3),
+    schedule_interval='0 10 * * *',
+    catchup=True,
+    default_args=DEFAULT_ARGS,
+    tags=['de2022'],
+    max_active_runs=1,
+)
+
+upload_files = PythonOperator(
```

Comment:
There's a ready-to-use LocalFilesystemToGCSOperator, so you reinvented a wheel :)
I haven't mentioned it in the HT description because I want the students to find it themselves.

Reply:
Дякую за Ваш коментар!
Я шукала, але чомусь не знайшла потрібний оператор, тому писала свою функцію. Врахую та трохи пізніше перероблю ДЗ. 
І cкріни також буду прикріпляти на далі в PR.